File: Pelikula.API/Model/Anketa/AnketaOdgovorKorisnikInsertRequest.cs
using System;

namespace Pelikula.API.Model.Anketa
{
    public class AnketaOdgovorKorisnikInsertRequest
    {
        public int AnketaOdgovorId { get; set; }

        public int KorisnikId { get; set; }
    }
}

File: Pelikula.MOBILE/lib/model/anketa/anketa_odgovor_korisnik_insert_request.dart
class AnketaOdgovorKorisnikInsertRequest {
  int? anketaOdgovorId;
  int? korisnikId;

  AnketaOdgovorKorisnikInsertRequest({this.anketaOdgovorId, this.korisnikId});

  factory AnketaOdgovorKorisnikInsertRequest.fromJson(
      Map<String, dynamic> json) {
    return AnketaOdgovorKorisnikInsertRequest(
      anketaOdgovorId: json['anketaOdgovorId'] as int,
      korisnikId: json['korisnikId'] as int,
    );
  }

  Map<String, dynamic> toJson() => {
        "anketaOdgovorId": anketaOdgovorId,
        "korisnikId": korisnikId,
      };
}
